realm/src/main/java/io/realm/internal/ColumnIndicies.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2014 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import io.realm.RealmObject;
+
+/**
+ * Utility class used to cache the mapping between object field names and their column indices.
+ */
+public class ColumnIndicies {
```

Comment:
I am not sure I understand that? 
This class is used by Realm.java, not the annotation processor, so it is not used to find getters and setters, but the column indexes for the fields of the RealmObject? These indexes are not known by the annotation processor so must be calculated at runtime. Also it is a map used by all RealmClasses not just a single one. So while it is true that you do two lookups, first to find the proper class data, then to find the index, it is two different lookups?